Remove 'Clinical Trial' from assay list

Bankso opened the pull request:
This term was causing an issue where the JSON-LD could not be parsed by schematic

vpchung commented on config.yml:
```diff
@@ -25,7 +25,7 @@ manifest:
 model:
   input:
-    location: '/Users/bzalmanek/Documents/csbc_schematic/schematic/tests/data/CSBCDataModel.jsonld'
+    location: '/Users/bzalmanek/Documents/csbc_schematic/schematic/tests/data/csbc.model.jsonld'
```
Should we rename the model JSON-LD to `mc2.model.jsonld`? 😅

brynnz22 replied:
Ah good point. I will change it now.

vpchung reviewed (approved):
Minor comment.

Bankso committed:
Remove 'Clinical Trial' from assay list

This term was causing an issue where the JSON-LD could not be parsed by schematic

Commit:
build jsonld
